src/Contracts/TerminalService.ts

```diff
@@ -0,0 +1,86 @@
+import WebSocket = require('isomorphic-ws');
```

Comment:
import WebSocket = require('isomorphic-ws');
import { Terminal } from 'xterm';
import * as fit from 'xterm/lib/addons/fit/fit';

please use "" for stirng, not single quote, to be same across all of our code. do not use Require, use notation as import * as foo from "Foo"